src/insecure/plaintext.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+const protobuf = require('protons')
+const protocol = '/plaintext/2.0.0'
+const handshake = require('it-handshake')
+const lp = require('it-length-prefixed')
+const PeerId = require('peer-id')
+const debug = require('debug')
+const log = debug('libp2p:plaintext')
+log.error = debug('libp2p:plaintext:error')
+const { UnexpectedPeerError, InvalidCryptoExchangeError } = require('libp2p-interfaces/src/crypto/errors')
+
+const {
+  Exchange,
+  KeyType
+} = protobuf(`
```

Comment:
should we isolate the protobuf in its own file? I think it is easier to have it because we can easily find the proto file and it does not add noise to the source code